news/windows_redirection.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+Fixed error when using redirection (e.g., >) on Windows. #2470
```

Comment:
@robdefriesse I think the checker is a little picky about the format in the news file. Hence the failures. The line must begin with an asterix (eg. `* Fi...` a). Also,you can remove the hashtag for the PR, that will not be rendered nicely anyway.

Reply:
Hopefully, that should fix the failures.